pom.xml

```diff
@@ -8,7 +8,7 @@
 
   <groupId>org.csanchez.jenkins.plugins</groupId>
   <artifactId>kubernetes</artifactId>
-  <version>1.1-SNAPSHOT</version>
+  <version>1.1-test-005</version>
```

Comment:
Will the maven release plugin like this?

Reply:
needs to be removed

Reply:
That was not intended! I need to get rid of it! Thanks!